packages/reflectable/example/example.dart

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2025, the Dart Team. All rights reserved. Use of this
```

Comment:
Alternatively you could make the example depend on the builder, and have its own pubspec.

I don't think this example is really helpful.

Reply:
Yes - You can include it in the workspace.